lib/urlwatch/jobs.py

```diff
@@ -358,4 +364,4 @@ def retrieve(self, job_state):
         from requests_html import HTMLSession
         session = HTMLSession()
         response = session.get(self.navigate)
-        return response.html.html
+        return (response.html.html, response.headers.get('Content-type', ''))
```

Comment:
Should there be a default fallback value for `response.headers.get()`? Maybe `text/html`?

Reply:
A typical web server would have that header. If it is missing, then the server is probably a custom script. Then maybe `text/plain`? or using [magic.from_buffer()](https://github.com/messense/magicfile#usage) to determine it?

Reply:
Yes, `text/plain` is probably also fine. However, how does that deal with headers like:

    Content-Type: text/html; charset=utf-8

Reply:
It's returned in a tuple and stored as `content_type`. Additionally, `text/html` is extracted and stored as `mime_type`.

Reply:
Oh, I forgot that empty `content_type` is also acceptable, and in that case `job.content_type` and `job.mime_type` are not created. Maybe that's the way to go, rather than guess?